### ISSUE TITLE
Registration page leads to 500 Internal Servor Error

### ISSUE THREAD

**Issue by pratik60** [getsentry/sentry#1460]
By default, when we initialize sentry config, this line is not there => 
SENTRY_ALLOW_REGISTRATION

But in this file sentry/web/frontend/accounts.py ? in register function we have
```
    if not (settings.SENTRY_ALLOW_REGISTRATION or request.session.get('can_register')):
```

As this condition isn't satisfied, I'm getting an internal server error on the registration page and the sentry logs for the backed are spiking like crazy.
```
django.conf in __getattr__
AttributeError: 'Settings' object has no attribute 'SENTRY_ALLOW_REGISTRATION'
```

Would have submitted a pr has well, but on /src/sentry/utils/runner.py you clearly have mentioned that this this is now deprecated. 

```
if hasattr(settings, 'SENTRY_ALLOW_REGISTRATION'):
warnings.warn('SENTRY_ALLOW_REGISTRATION is deprecated. Use SENTRY_FEATURES instead.', DeprecationWarning)
settings.SENTRY_FEATURES['auth:register'] = settings.SENTRY_ALLOW_REGISTRATION
```

Can we patch this up as required? If I did something wrong, please let me know. (Using Sentry 7.4.1)

**Comment by dcramer** [comment]
This is actually fixed in master. You can fix it in your local config by settings SENTRY_ALLOW_REGISTRATION = [True/False]

**Comment by dcramer** [comment]
Of note this was also fixed in 7.4.2
